.gitignore

```diff
@@ -1,4 +1,7 @@
 .DS_Store
 css/exercise_one/part-c/css/exercise_one.css.map
 css/exercise_one/part-d/css/exercise_one.css.map
+css/excercise_two/styles/styles.css.map
 *.map
+css/exercise_two/node_modules
+js/exercise_one/node_modules
```

Comment:
you need to remove the node modules from the cache, if not it won't be removed from here.
also you can remove the folder and when it comes back I believe it should be also ignored